app/uk/gov/hmrc/cdsreimbursementclaimfrontend/config/package.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend
+
+import configs.ConfigReader
+import configs.syntax._
+import play.api.Configuration
+
+package object config {
+
+  implicit class ConfigurationOps(val config: Configuration) extends AnyVal {
```

Comment:
This would be better placed inside your package object in the `upload` package rather than here unless you want to follow this pattern for grabbing the configuration for the whole of the service.

Reply:
Yes, I was thinking about the hole service which actually I did, except ViewConfig - it is pretty much doing the same thing. Also another reason why it is defined in the `config` package is that I need to reference it in the services package, because having reference from the `controller.fileupload` package is not good.

Reply:
There are specific files for the different sets of configs we introduce in the service: ViewConfig as you know is the config to drive the UI, EnrolmentConfig is for the enrolments. If you want to introduce a way to grab only the file upload config, create an object called FileUploadConfig and import that where required. That would be more consistent with the design that is there rather than a package object.